src/contract_wrappers/tokens/security_token_wrapper/common.ts

```diff
@@ -2265,6 +2319,25 @@ export default abstract class SecurityTokenCommon extends ERC20TokenWrapper {
     return bytes32ArrayToStringArray(await (await this.contract).getAllDocuments.callAsync());
   };
 
+  public signTransferData = async (params: SignTransferDataParams): Promise<string> => {
+    const [callerAddress, gtmAddresses, nonce] = await Promise.all([
+      this.getCallerAddress({}),
+      this.getModulesByName({ moduleName: ModuleName.GeneralTransferManager }),
+      this.getCurrentNonce({}),
+    ]);
+    const hash = getHashedTransferData(gtmAddresses[0], params.investorsData, params.validFrom, params.validTo, nonce);
```

Comment:
What if more than 1 GTM module attached?

Reply:
We're assuming in the SDK that there is only one GTM attached. Since data is stored in datastore contract, it doesn't makes any sense to have two GTM in the current implementation of our protocol.